core/templates/services/date-time-format.service.spec.ts

```diff
@@ -83,6 +84,12 @@ describe('datetimeformatter', () => {
         NOW_MILLIS - 365 * 24 * 60 * 60 * 1000)).toBe('11/21/13');
   });
 
+  it('should provide date time hour string', function() {
+    expect(
+      df.getLocaleDateTimeHourString(NOW_MILLIS)).toBe(
+      moment(new Date(NOW_MILLIS)).format('MMM D HH:mm A'));
```

Comment:
Best to give the output string, and not recompute it.

Reply:
Ok noted.